examples/simulation.py

```diff
@@ -0,0 +1,208 @@
+"""A minimum viable simulation"""
+# stdlib
+import argparse
+from typing import Any, Optional
+
+# external imports
+import numpy as np
+from numpy.random import Generator
+from stochastic.processes import GeometricBrownianMotion
```

Comment:
I am a wary about adding `stochastic` as a dependency for our repo. It seems popular (322 stars) but is maintained primarily by a single PhD student. I've been burned by this type of thing in the past. The [code](https://scipy-cookbook.readthedocs.io/items/BrownianMotion.html) to generate it from more core repos is pretty straight forward. What are your thoughts on doing it in-house using just scipy & numpy?